Resources/Prototypes/Hydroponics/seeds.yml

```
@@ -1336,6 +1337,36 @@
       Max: 10
       PotencyDivisor: 10
 
+- type: seed
+  id: ambrosiaHorribilis
+  name: seeds-ambrosiahorribilis-name
+  noun: seeds-noun-seeds
+  displayName: seeds-ambrosiahorribilis-display-name
+  plantRsi: Objects/Specific/Hydroponics/ambrosia_horribilis.rsi
+  packetPrototype: AmbrosiaHorribilisSeeds
+  productPrototypes:
+    - FoodAmbrosiaHorribilis
+  lifespan: 5
+  maturation: 10
+  production: 3
+  yield: 4
```

```suggestion
  yield: 3
```
should not be highter than the other two ones